### PR TITLE
Added help messages

### PR DESCRIPTION
**Pull request by HARDY8118**
Added some arguments checks and help messages to make usage a little easy.

### CODE REVIEW

File: db.sh [hunk 1]
```diff
@@ -2,51 +2,110 @@
...
-   sed -ri "/^"$key"\ .*/d" "$db"
+   if [[ -z "$1" ]];
+   then
+      echo "No database provided"
+   elif [[ "$1" == "--help" ]];
+   then
+      echo "Usage: db delete <database> <key>"
+   elif [[ -z "$2" ]];
+   then
+      echo "No key provided"
```
Comment (reddec) [inline db.sh:84]: (here, above, and bellow) Seems like 

```
echo "..."
exit 1
```

will remove the following `else` and allows user to rely on the exit code

File: db.sh [hunk 2]
```diff
@@ -2,51 +2,110 @@
...
 
 # Remove record by key
 delete () {
-   db=$1;
-   key=$(echo -n "$2" | base64 -w 0);
-   sed -ri "/^"$key"\ .*/d" "$db"
+   if [[ -z "$1" ]];
+   then
+      echo "No database provided"
+   elif [[ "$1" == "--help" ]];
```
Comment (reddec) [inline db.sh]: (here, above, and bellow) all commands require `<db>` as the first argument. Maybe we can move this check before "$@"

File: db.sh [hunk 3]
```diff
@@ -2,51 +2,110 @@
...
-"$@"
+if [[ -z "$1" || "$1" == "--help" ]];
+then
+   echo "Usage: db <method> <database> [arguments...]"
+else
+   if [[ "$1" =~ (get|list|last|put|delete) ]];
+   then
+      "$@"
+   else
+      echo "Usage: db <get|list|last|put|delete> <database> [arguments...]"
```
Comment (reddec) [inline db.sh]: What do you think if all help will be in thin place? Something like

```shell
echo <<<EOF
    Usage: $0 <get|list|last|put|delete> <database> [arguments...]

     get <db> [key]         - get single item
     put <db> <key> [value] - put single item
EOF
```

**Review by reddec — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!

File: db.sh (hunk 1 above)
Comment (HARDY8118) [reply db.sh:84]: That looks reasonable.
I will definitely work on that.

File: db.sh (hunk 2 above)
Comment (HARDY8118) [reply db.sh]: Great call.
It maked more sense here.

**Comment by HARDY8118** [comment]
Thank you for the suggestions.
I have made the changes and need your review on them.

**Review by reddec — APPROVED** [review]

**Comment by reddec** [comment]
@HARDY8118 thanks!

**Comment by HARDY8118** [comment]
Glad to help.
I found your idea of developing a database just based on bash functions really interesting and unique.

**Pull request merged**